# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/reftest/1961387-2-ref.html
# Commit: 6e6b5ae23a5c
# Full Hash: 6e6b5ae23a5cbe565ff56bf76f715acc17926549
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2025-05-09 04:15:00
# Regressor Bug: 1961387
# File Overlap Count: 3
# Description:
#   Bug 1961387 - Add a reftest with larger glyphs, for CBDT version of Noto Color Emoji. r=gfx-reviewers,lsalzman
#   
#   Without the scaling fix, the advance of the bitmap emoji glyph isn't
#   scaled according to the rendered size, and so the large glyph extends
#   beyond the expected rect.
# ==============================================================================

diff -r a884b6f401c3 -r 6e6b5ae23a5c gfx/tests/reftest/1961387-2-ref.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/reftest/1961387-2-ref.html	Thu May 08 17:42:11 2025 +0000
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<style>
+    div {
+        position: absolute;
+        writing-mode: vertical-lr;
+        font: 250px serif, Noto Color Emoji;
+    }
+    #mask {
+        color: transparent;
+        background: green;
+    }
+</style>
+
+<p>The emoji glyph should not be visible outside the green box:</p>
+
+<div id="mask">.&#x1f400;.</div>